src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,78 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Ui {
+
+    private final String lines = "--------------------------------------------------------------------------------------------";
+
+    /**
+     * Constructor for class Ui
+     */
+    public Ui() {
+    }
```

Comment:
As the constructor did not require any variables maybe you do not have to include it in your code